src/lists/list-item.jsx

```diff
@@ -407,6 +407,7 @@ const ListItem = React.createClass({
     const {
       innerDivStyle,
       style,
+      ...other,
```

Comment:
> I am seeing the same value

It's not destructured in the same way, we can't have the same list of properties.
Here is the one to use: https://github.com/callemall/material-ui/blob/master/src/lists/list-item.jsx#L136